tools/epicseg/epicseg_normalizecounts.xml

```diff
@@ -0,0 +1,66 @@
+<tool id="epicseg_normalizecounts" name="EpiCSeg - Normalize counts" version="0.1.0">
+	    <macros>
+		<import>macros.xml</import>
+	    </macros>
+	    <requirements>
+		<requirement type="package" version="1.0">epicseg</requirement>
+	    </requirements>
+	    <command detect_errors="exit_code"><![CDATA[
+
+		    #for $dataset in $datasets#
+              ln -s ${dataset.counts} `basename ${dataset.counts}` &&
+            #end for#
+
+            Rscript '${__tool_directory__}/epicseg.R' normalizecounts
+                --nthreads $nthreads
+
+            #for $dataset in $datasets#
+                --counts `basename ${dataset.counts}`
+            #end for# &&
+
+            rename "s/.dat/.txt/" *_norm.dat &&
+            rename 's/^/norm_/' *_norm.txt;
+	    ]]></command>
+	    <inputs>
+		<repeat name="datasets" title="Count matrix">
+		    <param type="data" name="counts" label="Count matrix"
+                   format="txt"
+
+                   help="Count matrix such as those produced by
+                         epicseg_getcounts. In case multiple count matrices
+                         are used, they must have the same marks and the same
+                         number of bins"/>
+		</repeat>
+		<param type="integer" name="nthreads"
```

Comment:
remove this one, a user does in general not know how many cores the cluster node has ...